data/json/construction.json

```
@@ -3637,8 +3637,8 @@
     "id": "constr_carpet_red",
     "group": "carpet_floor_red",
     "category": "DECORATE",
-    "required_skills": [ [ "fabrication", 1 ] ],
-    "time": "30 m",
+    "required_skills": [ [ "fabrication", 0 ] ],
+    "time": "1 m",
```

Per discussion I'd suggest this split the diff. Half an hour per segment is excessive, but a 5mx5m room also takes much longer than 25 mins to lay carpet. Especially if you're dropping the requirements to fab0, so you could easily have someone doing this who doesn't know how to read a tape measure.
```suggestion
    "time": "15 m",
```